src/pages/RegisterPage.js

```diff
@@ -27,7 +26,7 @@ export default function RegisterPage({setAuth}) {
         password
       }
 
-      const response = await fetch('http://localhost:5000/auth/register', {
+      const response = await fetch(`${process.env.REACT_APP_BASE_URL}/auth/register`, {
```

Comment:
Also one tip here, I forgot to write it last time. Nice change, but this can be even better. Imaging you are working with me on a project and I change the API contract so its not /auth/register, its now /user/register. Its good to have file with all this defined and just imported and used all over the project, this is not mandatory its just a tip for future

Reply:
yeah it's better